src/components/dropdown/positioning.coffee

```diff
@@ -1,6 +1,14 @@
 import { clamp } from 'utils/utils'
 
-export calculateDropdownPosition = (alignments, selectionRect, dropdownRect, windowRect, ddMaxHeight, scrollbarWidth) ->
+export calculateDropdownPosition = (align, selectionRect, dropdownRect, windowRect, ddMaxHeight, scrollbarWidth) ->
+  alignQuad = switch align
+    when 'up' then 'ltlb'
+    when 'down' then 'lblt'
+    when 'left' then 'ltrt'
+    when 'right' then 'rtlt'
+    else align
+
+  alignments = alignQuad.split('')
```

Comment:
Moved this inside the `position` calculation so this function now takes the `align` string as passed into the dropdown. This also means we can use the `direction` to save the position